src/test/cypress/cypress/integration/UserCard.spec.js

```diff
@@ -1204,10 +1203,10 @@ describe('User Card ', function () {
       usercard.selectRecipientForInformation('Control Center FR East');
       usercard.preview();
 
-      cy.get("#opfab-entity-recipients").find('span').should('have.length', 2);
+      cy.get("#opfab-entity-recipients").find('span').should('have.length', 3);
       usercard.checkEntityRecipientsInPreviewContains("Control Center FR North");
 
-      cy.get("#opfab-entity-recipients-for-information").find('span').should('have.length', 3);
+      cy.get("#opfab-entity-recipients-for-information").find('span').should('have.length', 5);
```

Comment:
Checking for the number of spans in this section is not relevant (lines 1206 and 1209)